src/Xamarin.Android.Tools.AndroidSdk/AndroidSdkInfo.cs

```diff
@@ -116,5 +116,28 @@ public bool IsPlatformInstalled (int apiLevel)
 		public string AndroidNdkHostPlatform {
 			get { return sdk.NdkHostPlatform; }
 		}
+
+		public IEnumerable<string> AllAndroidSdkPaths {
```

Comment:
should these be Obsoleted so that we are at least reminded we need to stop using them at some point?